src/theme/themeContext.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import { useContext} from 'react';
+import { ThemeProvider } from '@material-ui/core';
+import themeLight from 'src/theme/theme-light';
+import themeDark from 'src/theme/theme-dark';
+
+const Context = React.createContext(null);
```

Comment:
use `Context.displayName` https://reactjs.org/docs/context.html#contextdisplayname

Reply:
We need useState in ``setTheme(theme === themeLight ? themeDark : themeLight)`` like in  example
https://reactjs.org/docs/context.html#updating-context-from-a-nested-component ``app.js``
Also for controlling toggle icon and toggle tooltip. I need to do this way, because when click on logo it reset icon and tooltip but theme stay.

Reply:
others done